docs/client_tools/plugins/deadline.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Deadline plugin
+summary: Instructions for using and install the Deadline plugin.
+authors:
+    - Jesse Lehrman
+date: 2019-11-20
+---
+
+# Deadline plugin
+
+## Introduction
+Conductor is integrated into Deadline via a set of plugins and scripts. This toolset is responsible for sending jobs to Conductor and ensuring these jobs only run on Conductor instances. The uploading of dependencies and the download of the resulting data is handled by Conductor's upload and download daemons respectively.
+
+One of Conductor's underlying principles is to only spin-up an instance when it's needed by a specific job or task. This ensures that customers don't pay for idle machines. This is a shift from Deadline's (and most render farm manager's) paradigm that machines are running but idle until a job is assigned to them. This difference in paradigm is the main factor why Conductor wasn't implemented as a Deadline Cloud Plugin.
+
+Just as with any other instance in a Deadline managed farm, Conductor instances run the Deadline worker.
+
+*As of Deadline 10.1, the term **Slave** has been replaced with **Worker***
+
+## Installation
+
+![](../../image/download_icon.png) Download the *[plugin](https://github.com/ConductorTechnologies/deadline_plugin)* from Github
+
+*Steps 3-5 require Super User access in Deadline Monitor*
+
+1. Start *[Conductor uploader and downloader daemons]()*
+2. Copy plugins to the correct folders
+    1. `cp deadline_plugin/custom/scripts/Jobs/* -> <your_deadline_repo>/custom/scripts/Jobs/`
```

Comment:
Mine too.. not sure what to do. Using a code block didn't work. It was supposed to be pseudo-commands. I removed the 'cp'  to avoid confusion that it was platform specific.

Reply:
I think I fixed it. Let me know what you think.